signup.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 - 회원가입</title>
+    <link
+      rel="stylesheet"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="stylesheet" href="signlink.css">
+  </head>
+
+    <body>
+      <header>
+        <div class="bigLogo">
+            <a href="/">
+                <img id="panda"
+                 src="images/logopandal2x.png" alt="판다마켓">
+            </a>
+        </div>
+      </header>
+        <form class=" login">
+          <label>이메일</label>
+          <input
+            id="email"
```

Comment:
P2:
label과 input을 연결해주시면 label 클릭시 input으로 focus가 이동됩니다.
일반적인 동작이므로 연결해주시는것이 좋습니다.

```suggestion
          <label for='email'>이메일</label>
          <input
            id="email"
```